### PR TITLE
Species metadata

### PR DESCRIPTION
**Pull request by vincerubinetti**
Closes https://github.com/biothings/mygeneset.info/issues/26

- updates packages
- refactor request code
- add queries for species searching and getting "top species"
- add more icons
- add icons for all "top species"
- refactor/restyle species select
- species select now returns "top species" (ones included in the most genesets) when nothing typed in, and queries API when something typed in
- change species icon filenames to scientific, and load all with webpack, and dynamic match to query results
- refactor table component a bit
- style browse table better
- add slow option for tooltip
- add tooltip over species options and tags
- allow searching by taxon ids in species search
- fix geneset query bug from `q` to `species`

### CODE REVIEW

**Comment by ravila4** [comment]
A couple of improvements to the search UI:

1. Allow searching genesets by taxonomic id in the search box.

2. Provide total number of hits in the search results. The  current navigation returns 100 hits by default, and displays "1 to 10 of 100" 
    We may want to make it clear when 100 is not the full number of results.  Perhaps show a text saying "X number of results, Showing top 100".  Perhaps we should also allow the user to customize this number. Would there be cases when a user might want to get more that 100 results?

File: src/api/genesets.ts [hunk 1]
```diff
@@ -19,6 +18,6 @@ export const search = async (query?: string, species?: string[]): Response => {
   if (species?.length) params.set("q", species.join(","));
```
Comment (ravila4) [inline src/api/genesets.ts:18]: The parameter to search here should be "species" instead of "q"

**Review by ravila4 — COMMENTED** [review]
I found a bug with the species search. When selecting an organism, no results are shown. 
Screenshot from 2021-02-26 14-32-20
The issue seems to be this query:
Screenshot from 2021-02-26 14-32-43

The correct query should be: 	https://mygeneset.info/v1/query?species=6239&fields=*&size=100

**Comment by vincerubinetti** [comment]
Haven't figured out best way to do 2., but merging because I want to see if the CNAME fixes the domain/redirect issue.

**Pull request merged**